frontend/src/features/workspace/components/DuplicateFormModal/DupeFormWizardProvider.tsx

```diff
@@ -53,7 +54,12 @@ export const useDupeFormWizardContext = (): CreateFormWizardContextReturn => {
       responseMode: containsMyInfoFields
         ? FormResponseMode.Email
         : FormResponseMode.Encrypt,
-      title: makeDuplicateFormTitle(previewFormData.form.title, dashboardForms),
+      title: makeDuplicateFormTitle(
+        isTemplate
+          ? `Template_${previewFormData.form.title}`
```

Comment:
not about your PR, but omggg why are we still prefixing with "Template_" so gross :(

Reply:
@staceytan1998 any comments HAHA

Reply:
we'll change this to [Template] after we allow special characters in the form title

Reply:
[3724a12](https://github.com/opengovsg/FormSG/pull/5377/commits/3724a12d3e816e7ee2988d401dfb0cf0915f1d72) updated yay